dstep/driver/Application.d

```diff
@@ -48,16 +46,22 @@ class Application : DStack.Application
 
     protected override void run ()
     {
+        derr.writefln(arguments.rawArgs);
         handleArguments;
 
         if (arguments.argument.input.hasValue)
         {
-            inputFiles ~= arguments.argument.input;
-            startConversion(inputFiles.first);
+            auto inputFiles = arguments.argument.input.values[1 .. $];
+            foreach(string fileName; inputFiles)
+            {
+                auto conversionTask = task!startParsingFile(language, argsToRestore, fileName, true, arguments, compiler, compilerArgs);
+                conversionTask.executeInNewThread();
+            }
+            startParsingFile(language, argsToRestore, arguments.argument.input.values[0], false, arguments, compiler, compilerArgs);
```

Comment:
I thought it would reduce total number of threads spawned. Also, if only 1 file is parsed, no need to create extra thread.
Should I include first file in new thread?